File: inutile/xx.php
<?php
$data = [];
for ($i = 0; $i < 10; $i++) {
    $data[] = ["id" => $i, "value" => rand(0, 99)];
}

$json_data = json_encode($data, JSON_PRETTY_PRINT);
file_put_contents('data.json', $json_data);

echo "Data generated and saved to data.json\n";
?>

File: inutile/xx.rs
use rand::Rng;
use serde::Serialize;
use std::fs::File;
use std::io::Write;

#[derive(Serialize)]
struct DataItem {
    id: u32,
    value: u32,
}

fn main() {
    let mut rng = rand::thread_rng();
    let data: Vec<DataItem> = (0..10)
        .map(|i| DataItem {
            id: i,
            value: rng.gen_range(0..100),
        })
        .collect();

    let json_data = serde_json::to_string_pretty(&data).unwrap();
    let mut file = File::create("data.json").unwrap();
    file.write_all(json_data.as_bytes()).unwrap();

    println!("Data generated and saved to data.json");
}